src/main/resources/db/changeLog.xml

```diff
@@ -178,4 +178,36 @@
         <!-- Update milestone original dates to a default value -->
         <sql>UPDATE milestone SET originalstartdate = startdate, originalenddate = enddate</sql>
     </changeSet>
+
+    <changeSet id="taskparams" author="daniil smirnov">
+        <addColumn tableName="task">
+            <column name="userrole" type="varchar(255)"/>
+            <column name="estimatedduration" type="varchar(255)"/>
+            <column name="estimatedreadiness" type="varchar(255)"/>
```

Comment:
This is ok, but it could quite safely be made shorter, for instance, max length of 191.